caddyhttp/rewrite/rewrite.go

```diff
@@ -230,6 +228,10 @@ func (r *ComplexRule) matchExt(rPath string) bool {
 	return true
 }
 
+func newReplacer(r *http.Request) httpserver.Replacer {
+	return httpserver.NewReplacer(r, nil, "")
+}
+
```

Comment:
I think this is only used once (at least, that's what a quick Cmd+F on this page tells me) - maybe not needed then?